addons/website_sale/models/product_product.py

```diff
@@ -69,8 +69,11 @@ def _compute_base_unit_name(self):
     @api.depends('product_tmpl_id.website_url', 'product_template_attribute_value_ids')
     def _compute_product_website_url(self):
         for product in self:
-            attributes = ','.join(str(x) for x in product.product_template_attribute_value_ids.product_attribute_value_id.ids)
-            product.website_url = "%s#attribute_values=%s" % (product.product_tmpl_id.website_url, attributes)
+            pavs = product.product_template_attribute_value_ids.product_attribute_value_id
+            url = product.product_tmpl_id.website_url
+            if pavs:
+                url = "%s#attribute_values=%s" % (url, ','.join(str(x.id) for x in pavs))
```

Comment:
Clearer?
```suggestion
            url = product.product_tmpl_id.website_url
            if pavs := product.product_template_attribute_value_ids.product_attribute_value_id:
                pav_ids = [str(pav.id) for pav in pavs]
                url = f'{url}#attribute_values={",".join(pav_ids)}'
```

Reply:
Mhm, I don't find it clearer, but not really worse either, so no strong opinion, suggestion applied.